### ISSUE TITLE
Red highlighting of menu items (is this a bug?)

### ISSUE THREAD

**Issue by PeteBeet** [Open-Shell/Open-Shell-Menu#1694]
### Describe the bug

Seen in 4.4.190 and 4.4.191

When clicking Start, certain items are highlighted in red, either in main popup, programs or apps.

### Area of issue

Start menu

### To reproduce

Click Start
See highlighted items in main menu, Programs and/or Apps.

![screenshot menu highlighting 4-4-191](https://github.com/Open-Shell/Open-Shell-Menu/assets/144205800/8d20ca13-bd3a-4ece-a2c6-a1d26b2f1f46)

### Expected behavior

I didn't expect to see these highlighted items.

Not sure if this is due to a bug, but can't see a logical reason why these are highlighted.

### Open-Shell version

4.4.191

### Windows version

Windows 10 Pro 22H2

**Comment by ge0rdi** [comment]
Most likely those are _newly_ installed programs/apps and you have either `Highlight newly installed programs` or `Highlight newly installed apps` setting enabled.

Can you confirm?

**Comment by PeteBeet** [comment]
@ge0rdi Indeed. That's why I asked if it's a bug. I've never seen this setting before, possibly because of the color of the highlight. Apparently both of those boxes are checked by default. Clearing them is just what I want. Thank you.
